templates/repo/diff/box.tmpl

```diff
@@ -198,9 +200,9 @@
 								<div id="diff-rendered-{{$file.NameHash}}" class="file-body file-code {{if $.IsSplitStyle}}code-diff-split{{else}}code-diff-unified{{end}} gt-overflow-x-scroll">
 									<table class="chroma gt-w-100">
 										{{if $isImage}}
-											{{template "repo/diff/image_diff" dict "file" . "root" $ "blobBase" $blobBase "blobHead" $blobHead}}
+											{{template "repo/diff/image_diff" dict "file" . "root" $ "blobBase" $blobBase "blobHead" $blobHead "stBase" $stBase "stHead" $stHead}}
```

Comment:
When used as template parameter, I think it needs a full/clear name, but not `st`.

Reply:
Done in https://github.com/go-gitea/gitea/pull/27267/commits/4c0212ae0d75f20b6d05684c112167a9ef1f2998.